lib/provider/vpc_data_types.go

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Container Service, 5737-D43
+ * (C) Copyright IBM Corp. 2018 All Rights Reserved.
+ * The source code for this program is not  published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+package provider
+
+// VPC specific	parameters
+type VPCVolume struct {
+	Href              string              `json:"href,omitempty"`
```

Comment:
I don't think `Href` is useful for the user, can we keep only variable which are useful for user as this is user(client) facing struct.

same with CRN, Generation and Profile

Reply:
NEEDED